pkg/products/cloudresources/reconciler.go

```diff
@@ -357,8 +357,9 @@ func (r *Reconciler) reconcileCIDRValue(ctx context.Context, client k8sclient.Cl
 		return err
 	}
 
-	if !ok {
-		return nil
+	//!ok means the param wasn't found so we want to default rather than return
+	if !ok || cidrValue == "" {
+		cidrValue = "10.1.0.0/16"
```

Comment:
Defaulting to a static range could run into issues, as this CIDR can not overlap with Cluster CIDR. There is logic in CRO to get the Cluster CIDR, we should set the default there or export the logic so we can insure there is no overlap and the CIDR is valid.

Reply:
This could happen in the case of a customer provided CIDR also though right?

Reply:
yep, that could also happen, the validation on the form is only regex and ensures it is a valid CIDR. There is no other validation provided via CS.

Reply:
CRO will report back if the CIDR range is not suitable so I think we're covered there. We just need to find a way to provide some sensible default via rhoam.